tests/python_tests/test_vlm_pipeline.py

```diff
@@ -921,10 +924,31 @@ def test_vlm_npu_no_image():
     )
 
 
+@pytest.mark.skipif(
+    sys.platform == "darwin" or platform.machine() in ["aarch64", "arm64", "ARM64"],
+    reason="NPU plugin is available only on Linux and Windows x86_64",
+)
+def test_vlm_npu_multiple_images(cat_tensor: openvino.Tensor, handwritten_tensor: openvino.Tensor):
+    models_path = _get_ov_model(MODEL_IDS[0])
+    properties = {
+        "DEVICE_PROPERTIES": {
+            "NPU": {"NPUW_DEVICES": "CPU", "NPUW_ONLINE_PIPELINE": "NONE", "MAX_PROMPT_LEN": 2048}
+        }
+    }
+
+    ov_pipe = VLMPipeline(models_path, "NPU", config=properties)
+
+    generation_config = _setup_generation_config(ov_pipe)
```

Comment:
Optionally, move to a fixture similar to CPU to avoid multiple constructions. This should improve the stability

Reply:
@Wovchena Done. Please re-review